.circleci/config.yml

```diff
@@ -45,7 +45,8 @@ jobs:
            name: Run tests
            command: |
              . .venv/bin/activate
-             python -m pytest
+             python -m pytest image/golem_blender_app/tests
```

Comment:
Unfortunately there is no `pytestrc`. Besides, I cannot test this locally. Neither `pytest` or `python -m pytest` commands work for me, since the `tests` package is not really installed

Reply:
checked the docs and it would be `pytest.ini`

but lets leave it when its not easy to reproduce locally